src/components/OrganizationsDropdown.tsx

```diff
@@ -43,9 +47,24 @@ export default function OrganizationsDropdown(): JSX.Element {
     }
   };
 
+  const onOrgCreated = useCallback(
```

Comment:
Can you add this to the `OrganizationProvider`? Maybe you can just add a prop to `saveOrganization` like `redirectAndNotify`

Reply:
Happy to. There are actually a few callback patterns I would like to use for ApplicationProvider too. So be on the look out for this PR or next!